src/AcceptanceTests/Helpers/ApprenticeshipEntityExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.Extensions.Internal;
+using SFA.DAS.Funding.ApprenticeshipEarnings.Domain.Services;
+using SFA.DAS.Funding.ApprenticeshipEarnings.DurableEntities;
+using SFA.DAS.Funding.ApprenticeshipEarnings.DurableEntities.Models;
+
+namespace SFA.DAS.Funding.ApprenticeshipEarnings.AcceptanceTests.Helpers;
+
+internal static class ApprenticeshipEntityExtensions
+{
+    internal static ApprenticeshipEpisodeModel GetCurrentEpisode(this ApprenticeshipEntity apprenticeship, ISystemClockService systemClock)
```

Comment:
There are 2 test files for getCurrentEpisode which are exactly the same and a very similar one in the main code, could we reduce the code duplication

Reply:
sorted